packages/salesforcedx-test-utils-vscode/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@salesforce/salesforcedx-test-utils-vscode",
+  "displayName": "SFDX Test Utilities for VS Code",
+  "description": "Provides test utilies to interface the SFDX libraries with VS Code",
+  "version": "45.7.0",
+  "publisher": "salesforce",
+  "license": "BSD-3-Clause",
+  "categories": [
+    "Other"
+  ],
+  "dependencies": {
+    "@salesforce/salesforcedx-utils-vscode": "45.7.0",
+    "shelljs": "0.8.3"
+  },
+  "devDependencies": {
```

Comment:
Not part of these changes but I think we might be able to get rid of some of the `devDependencies` on `salesforcedx-utils-vscode/package.json`, which are very similar to the ones here.